ui/src/components/NavigationBar/MainMenu.js

```diff
@@ -173,9 +172,17 @@ const MainMenu = ({ login, onClose }) => {
                 >
                   <Button marginBottom="5" width="100%">
                     <LinkOverlay href="/login" onClick={onClose}>
-                      Login or Sign up
+                      LOGIN
                     </LinkOverlay>
                   </Button>
+                  <Button
+                    onClick={onClose}
+                    marginBottom="5"
+                    width="100%"
+                    style={{ WebkitMarginStart: '0', marginInlineStart: '0' }}
+                  >
+                    <LinkOverlay href="/signup">SIGN UP</LinkOverlay>
+                  </Button>
```

Comment:
<img width="176" alt="image" src="https://user-images.githubusercontent.com/24607458/183415466-eee5d02f-c636-4e44-8948-24e2a0294897.png">
the signup button should be the same color as the navbar when not hovered on.

Reply:
actually saw diff versions on the figma, if that the one cxo decided on then no need to change

Reply:
> actually saw diff versions on the figma, if that the one cxo decided on then no need to change

It's the one top, the one with blue and dark blue when hovered on.